src/SolAR3DPointsViewerOpengl.cpp

```diff
@@ -84,6 +86,27 @@ xpcf::XPCFErrorCode SolAR3DPointsViewerOpengl::onConfigured()
     m_resolutionX = m_width;
     m_resolutionY = m_height;
 
+    if (m_pointsColorFromClassLabel>0 && !m_classLabelColorMapPath.empty()) {
```

Comment:
nit: add log message if file path is not filled.
Ex (you can change message/error code)
```
if (m_pointsColorFromClassLabel)
{
  if( m_classLabelColorMapPath.empty() )
  {
    LOG_ERROR("Property classLabelColorMapPath is not defined and is needed when pointsColorFromClassLabel is > 0");
    return xpcf::XPCFErrorCode::_FAIL;
  }
```

Reply:
done